### PR TITLE
Proposal for Refactoring

### PR DESCRIPTION
**Pull request by SarahKay99**
I've changed the folder structure to allow for chapters to be sorted into courses/modules (courses -> *coursename* -> chapters) and demonstrated the structure with a new side-menu on the left. This folder structure will be compatible with the Modules page, drawn on Figma, which displays each course. Please share your thoughts on the new back-end.

Later, we may need to create another dimension (**category** -> course -> *coursename* -> chapters) called category, so that we can sort courses into categories. 

**Work to be done:**

- Fix chapter loading. Since I've changed the file paths, there are problems loading the chapters onto the UI.
- Fix routing. Instead of oceanacademy.io/chapter-1 it should be oceanacademy/modulename/chapter1.
- CSS restyling of the left-hand menu bar.

### CODE REVIEW

File: src/frontend/src/app/App.components/Drawer/ChapterDrawer.controller.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    hideMenuDrawer,
+    showOcean101Drawer,
+    hideOcean101Drawer,
+    showBusinessDrawer,
+    hideBusinessDrawer,
+    showOutreachDrawer,
+    hideOutreachDrawer,
+    showC2DDrawer,
+    hideC2DDrawer
+} from './Drawer.actions'
```
Comment (oceanByte) [inline src/frontend/src/app/App.components/Drawer/ChapterDrawer.controller.tsx:14]: Can't you use something like "toggleXYZDrawer" and use a boolean then?

**Review by oceanByte — COMMENTED** [review]
We are using a git-flow approach. Please create a merge request for the `development` branch.

**Comment by oceanByte** [comment]
Please resolve all conflicts before any further review.

> Conflicting files
> src/frontend/src/app/App.components/Drawer/Drawer.actions.tsx
> src/frontend/src/app/App.components/Drawer/Drawer.controller.tsx
> src/frontend/src/app/App.components/Drawer/Drawer.view.tsx
> src/frontend/src/app/App.components/Hamburger/Hamburger.controller.tsx
> src/frontend/src/app/App.components/Header/Header.view.tsx
> src/frontend/src/app/App.controller.tsx
> src/frontend/src/reducers/chapterDrawer.ts
> src/frontend/src/reducers/index.ts
> src/frontend/src/reducers/loginDrawer.ts

**Comment by SarahKay99** [comment]
Added animated waves

**Comment by SarahKay99** [comment]
Making a new PR which uses a lot of these changes, pls look there instead 👍🏼

**Pull request closed without merging**